src/Sulu/Bundle/SearchBundle/Tests/Functional/Controller/SearchControllerTest.php

```diff
@@ -155,6 +155,46 @@ public function provideSearch()
                     'total' => 2,
                 ],
             ],
+            [
```

Comment:
added this test case that page 1 and page 2 is tested to get the correct result.